### PR TITLE
Update joomla-image-select.css

### PR DESCRIPTION
**Pull request by hitesh-coder**
Pull Request for Issue #31909.

### Steps to reproduce
Content > Article
create a new article
click on cms-content dropdown select image
Click on any image additional data should popup

### Summary of Changes
added a different **background color** and also added **padding** and **border-radius** which i thinks looks good

### Expected result AFTER applying this Pull Request
This should make the necessary changes required for the desired styling

### CODE REVIEW

**Comment by infograf768** [comment]
Please post screenshots Before and After pr.

**Comment by hitesh-coder** [comment]
# Before

Screenshot 2021-03-18 at 5 59 36 PM


# After
Screenshot 2021-03-18 at 4 11 41 PM



@infograf768

**Comment by dgrammatiko** [comment]
Can you please also  style the summary element (to look like a button or an accordion header)?

**Comment by hitesh-coder** [comment]
Updated as you asked @dgrammatiko 

#BEFORE
Before

#AFTER
Screenshot 2021-03-19 at 3 26 28 PM

**Comment by drmenzelit** [comment]
Can you use the same border-radius as in other places on template?

You have to check your PR, there are more changed files as needed. And the tests are failing (you can click on "Details" to see what is causing the problem)
grafik

**Comment by drmenzelit** [comment]
The css files should not be commited

**Comment by dgrammatiko** [comment]
@hitesh-coder sorry your changes **need to be applied only** on this file https://github.com/joomla/joomla-cms/blob/4.0-dev/build/media_source/system/css/fields/joomla-image-select.css

**Comment by hitesh-coder** [comment]
@dgrammatiko but you asked to change summary element as accordion  header and that should be done in respective file
so should i now revert all my changes and apply as before

**Comment by brianteeman** [comment]
@hitesh-coder the problem is that your change is global and not just on this one element

File: build/media_source/system/css/fields/joomla-image-select.css [hunk 1]
```diff
@@ -1,6 +1,8 @@
 joomla-field-mediamore details {
   position: absolute;
-  background: #ffffff;
+  background: #eee;
+  padding: 1rem;
+  border-radius: 1rem;
```
Comment (dgrammatiko) [inline build/media_source/system/css/fields/joomla-image-select.css]: Please also add you previous changes, eg
```css
joomla-field-mediamore summary {
   padding: 1rem;
   background-color: #e3e3e3;
   border-top-left-radius: 1rem;
   border-top-right-radius: 1rem;
}
```

**Comment by hitesh-coder** [comment]
## AFTER
Screenshot 2021-03-19 at 7 50 16 PM

File: build/media_source/system/css/fields/joomla-image-select.css [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
 }
+
+joomla-field-mediamore summary {
+  padding: 1rem;
+  background-color: #e3e3e3;
+  border-radius: 0.25rem;
+  font-weight: 500;
+}
+
+joomla-field-mediamore details .form-group{
```
Comment (Quy) [inline build/media_source/system/css/fields/joomla-image-select.css]:
```suggestion
joomla-field-mediamore details .form-group {
```

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on b5f790af136cbe26aa252e23843e83960900e5b5<br /><br />Better with this styling. Should it have a border?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32728">issues.joomla.org/tracker/joomla-cms/32728</a>.</sub>

**Comment by meinhoonharsh** [comment]
I have not tested this item.<br /><br />I tried to Test this, but when I Apply Patch, I can't see any change.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32728">issues.joomla.org/tracker/joomla-cms/32728</a>.</sub>

**Comment by drmenzelit** [comment]
@meinhoonharsh you can't test this using the patch tester, because the PR changes a scss file and for this you need to run npm run build:css to work

**Comment by hitesh-coder** [comment]
hey @ceford thanks for testing but could you please elaborate on the thing you mention I would love to make it better

**Comment by infograf768** [comment]
@hitesh-coder 

add a border
```
joomla-field-mediamore details {
    position: absolute;
    background: #f5f5f5;
    border-radius: .25rem;
    border: 1px solid #c9c9c9; // add this
}
```
to get

Screen Shot 2021-03-29 at 09 14 35

**Comment by meinhoonharsh** [comment]
> @meinhoonharsh you can't test this using the patch tester, because the PR changes a scss file and for this you need to run npm run build:css to work

Thank You @drmenzelit , I will make sure to do this from now

**Comment by meinhoonharsh** [comment]
I have tested this item ✅ successfully on b5f790a
### **Before:**
image
### **After:**
image

**Comment by hitesh-coder** [comment]
hey @meinhoonharsh I have also added a border in my latest commit

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on 934c082a906bd3c75ad0e543a8b024f952a8d0b5<br /><br />Yes - really nice with a border!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32728">issues.joomla.org/tracker/joomla-cms/32728</a>.</sub>

**Comment by meinhoonharsh** [comment]
I have tested this item :white_check_mark: successfully on 934c082a906bd3c75ad0e543a8b024f952a8d0b5<br /><br />I have Successfully tested this. It looks good in order to comparision with previous one
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32728">issues.joomla.org/tracker/joomla-cms/32728</a>.</sub>

**Comment by infograf768** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32728">issues.joomla.org/tracker/joomla-cms/32728</a>.</sub>

**Comment by drmenzelit** [comment]
@hitesh-coder why did you close this PR?

**Comment by hitesh-coder** [comment]
@drmenzelit I mistakenly deleted the branch while exploring GitHub feature. can I make another PR?

**Comment by richard67** [comment]
@hitesh-coder No need to make a new PR. I've re-opened this so it can be merged. This can be done even when you branch was deleted, as far as I know.

**Comment by drmenzelit** [comment]
Thanks!

**Pull request merged**